### PR TITLE
Snug bugs unrelated to and never been inside of a rug

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

File: tools/splat_ext/pm_map_data.py [hunk 1]
```diff
@@ -1,9 +1,8 @@
+from math import ceil
 import os, sys
 from pathlib import Path
-from typing import List
 from segtypes.n64.segment import N64Segment
 from util.n64.Yay0decompress import Yay0Decompressor
-from util.color import unpack_color
```
Comment (nanaian) [inline tools/splat_ext/pm_map_data.py:6]: Why this change?

**Review by nanaian — CHANGES_REQUESTED** [review]
Question about a change related to `unpack_color` because now we are duplicating code

File: tools/splat_ext/pm_map_data.py (hunk 1 above)
Comment (ethteck) [reply tools/splat_ext/pm_map_data.py:6]: it's not in splat anymore. Eventually pigment/n64img will export these functions for others to use. 

For now I'd rather just copy the code than do more and more path hacking, which has already gotten really ugly. It makes it easy to identify all the clones, anyway

Comment (nanaian) [reply tools/splat_ext/pm_map_data.py:6]: Fair enough, once pigment gets it we should import it from there yes

**Review by nanaian — APPROVED** [review]
